### PR TITLE
A-1204870316844980

### PR DESCRIPTION
**Pull request by owlsua**
# :speech_balloon: Description 

Add transactions flow for the ML wallet

Created a new screen to be the second step of the account creation user flow.

### More

- Update Send transaction interface for the ML wallet
- Improving unit tests

# :camera: Screenshots
Screenshot 2023-08-18 at 18 14 51

Screenshot 2023-08-18 at 18 15 42
Screenshot 2023-08-18 at 18 16 11
Screenshot 2023-08-18 at 18 16 51

# :clipboard: Checklist:

- [x] I have named my branch as `A-[id of Asana task]`
- [x] I have set the title of my PR as `A-[id of Asana task]: [short description]`
- [x] My changes passed successfully by `prettier` check
- [x] My changes passed successfully by `lint` check
- [x] My changes kept the previous test coverage rate
- [x] I have added enough tests for my new feature/bugfix
- [x] I have set at least one person to review this PR
- [x] I have set myself as the assignee of this PR
- [x] I have set at least one label to this PR

### CODE REVIEW

File: src/components/composed/FeeField/FeeField.js [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
 }) => {
+  const { walletType } = useContext(AccountContext)
   const effectCalled = useRef(false)
   const [options, setOptions] = useState([])
   const [inputValue, setInputValue] = useState(0)
   const [radioButtonValue, setButtonValue] = useState(undefined)
   const [timeToFirstConfirmations, setTimeToFirstConfirmations] =
     useState('15 minutes')
   const [estimatedFees, setEstimatedFees] = useState([])
+  const feeType = walletType.name === 'Mintlayer' ? 'ML' : 'sat/B'
```
Comment (renie) [inline src/components/composed/FeeField/FeeField.js:50]: @muursh What is going to be called the smallest part of ML? Satoshis as well? How could we express the `sat/B` for ML fees?

**Review by renie — CHANGES_REQUESTED** [review]
Great Job! Very simple considerations.

File: src/components/composed/FeeField/FeeField.js (hunk 1 above)
Comment (muursh) [reply src/components/composed/FeeField/FeeField.js:50]: So far in the core codebase we call them atoms as we don't have a better name. For now we can call them atoms and update it once there's a real term

Comment (renie) [reply src/components/composed/FeeField/FeeField.js:50]: So @owlsua , lets replace `ML` by `atoms/B`

**Review by renie — APPROVED** [review]

**Pull request merged**